src/inmanta/parser/plyInmantaLex.py

```diff
@@ -132,46 +130,22 @@ def t_JCOMMENT(t: lex.LexToken) -> None:  # noqa: N802
     pass
 
 
-def t_begin_mls(t: lex.LexToken) -> lex.LexToken:
-    r'["]{3}'
-    t.lexer.begin("mls")
-    t.type = "MLS"
-
-    lexer = t.lexer
-
-    end = lexer.lexpos - lexer.linestart + 1
-    (s, e) = lexer.lexmatch.span()
-    start = end - (e - s)
-
-    t.value = LocatableString("", Range(lexer.inmfile, lexer.lineno, start, lexer.lineno, end), lexer.lexpos, lexer.namespace)
-
-    return t
-
-
-def t_mls_end(t: lex.LexToken) -> lex.LexToken:
-    r'.*["]{3}'
-    t.lexer.begin("INITIAL")
-    t.type = "MLS_END"
-    value = t.value[:-3]
-
+def t_MLS(t: lex.LexToken) -> lex.LexToken:
+    r'"{3}([\s\S]*?)"{3}'
```

Comment:
What's the difference between `[\s\S]` and `.`? It looks like you addressed the concern @Hugo-Inmanta raised but could you add a test case that explicitly tests this?

Reply:
apparently `.` doesn't match newlines and `[\s\S]` does. I'll add a test

Reply:
I see, thanks.